src/widgets/graph_widget.h

```diff
@@ -67,4 +67,16 @@ enum PerformanceRatingDetailsWidgets {
 	WID_PRD_COMPANY_LAST  = WID_PRD_COMPANY_FIRST + MAX_COMPANIES - 1, ///< Last company.
 };
 
+/** Widget of the #IndustryProductionGraphWindow class. */
+enum IndustryProductionGraphWidgets {
```

Comment:
Remember to run the scripts to update the script API with widget numbers, in the src/script/api/ directory.

Reply:
And the run order is generate_widget then squirrel_export

Reply:
Fixed in 889a104.